### PR TITLE
decompile MI_dma_card

### PR DESCRIPTION
**Pull request by red031000**
also change m2ctx to not include global.h by default (this should be included in each file)

### CODE REVIEW

File: lib/include/nitro/mi.h [hunk 1]
```diff
@@ -1,12 +1,13 @@
-#ifndef NITRO_MI_H_
-#define NITRO_MI_H_
+#ifndef POKEHEARTGOLD_MI_H
```
Comment (PikalaxALT) [inline lib/include/nitro/mi.h:1]: revert

File: lib/include/nitro/mi/dma.h [hunk 2]
```diff
@@ -1,34 +1,43 @@
-#ifndef NITRO_MI_DMA_H_
-#define NITRO_MI_DMA_H_
+#ifndef POKEHEARTGOLD_MI_DMA_H
```
Comment (PikalaxALT) [inline lib/include/nitro/mi/dma.h:1]: revert

File: lib/include/nitro/mi/dma_card.h [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#ifndef POKEHEARTGOLD_MI_DMA_CARD_H
```
Comment (PikalaxALT) [inline lib/include/nitro/mi/dma_card.h:1]:
```suggestion
#ifndef NITRO_MI_DMA_CARD_H
```

**Review by PikalaxALT — CHANGES_REQUESTED** [review]

File: lib/include/nitro/mi.h (hunk 1 above)
Comment (red031000) [reply lib/include/nitro/mi.h:1]: why?

Comment (PikalaxALT) [reply lib/include/nitro/mi.h:1]: because these are not gamefreak headers

Comment (red031000) [reply lib/include/nitro/mi.h:1]: I am aware of that, but as far as I know GF didn't use the POKEHEARTGOLD_<file>_H scheme either, we use it for every other file in the repo, I'm unsure why not here

Comment (red031000) [reply lib/include/nitro/mi.h:1]: if strictly necessary I can go through and change it to NITRO_MI_H (or NITROSDK_MI_H) but I'd need to change the headers in diamond too

**Review by PikalaxALT — APPROVED** [review]
Tentatively approving, though would prefer the nitro header guards to retain the trailing underscore.

**Review by adrienntindall — APPROVED** [review]

**Pull request merged**